### ISSUE TITLE
Add notice/log entry for missing ffmpeg - feature request

### ISSUE THREAD

**Issue by prabbit237** [Unmanic/unmanic#319]
Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)? By filing an Issue, you are expected
to comply with it, including treating everyone with respect:

Remember that these are some guidelines, use your best judgement.

### Description

[Description of the issue]
Missing ffmpeg causes failures without any obvious reason for failure.
https://github.com/Unmanic/unmanic/issues/305

### Steps to Reproduce

1. [First Step]
Install Unmanic

2. [Second Step]
Remove ffmpeg (or start with a freshly-setup Debian computer.)

3. [Third Step]
Add and configure library and add files.

4. [and so on...]
Scratch head as to why nothing is happening.

**Expected behavior:** [What you expect to happen]
Files get converted or an error shows as to why they don't.

**Actual behaviour:** [What actually happens]
I get a blank stare from Unmanic as it sits and does nothing.

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

### Versions

The version can be found at the footer of the WebUI. Also, please include the OS details and what version you are running
(e.g. `cat /etc/os-release`).

### Installation Method Used

Manually installed and run or run through Docker?

### Additional Information

Any additional information, configuration or data that might be necessary to
reproduce the issue.

Basically it'd be helpful if the log would say something like:
Starting scan...
Ffmpeg not found
Ending scan...
